app/controllers/users_controller.rb

```diff
@@ -49,7 +49,7 @@ def create
   private
 
   def set_user
-    @user = User.find(params[:id])
+    @user = current_user
   rescue ActiveRecord::RecordNotFound
```

Comment:
With the current logic, you can remove the rescue for ``ActiveRecord::RecordNotFound ``